integration_tests/features/steps/compare_images.py

```diff
@@ -58,4 +58,3 @@ def step_impl(context, output):
         assert exit_status == 0, "Files did not match with the correct output"
     finally:
         os.chdir(orig_dir)
-        shutil.rmtree(context.temp_dir)
```

Comment:
This wasn't always deleting files (ie when builds are interrupted, etc). I moved the removal of tmp files to Jenkins. Now tmp files are deleted before running run.sh and hence Jenkins will not run out of tmp space from tmp files, and we can debug easier by seeing the output files (since we don't delete them after the r=tests are ran).

Reply:
Why are builds interrupted? Manually I suppose? And/or does that include Jenkins detecting that a new build is being started so it can cancel currently running ones?

Can we retain the removal in the behavior tests so that they can be run outside of/without Jenkins?

Reply:
Sure, I'll have to test why not all tmp files are being removed. There's usually a couple left over.